src/test/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# JAWSDB database properties for register back end
```

Comment:
Did you get the email from git about exposing Database secret keys as well? :)

Reply:
Yes, initially it didn't occur to me that putting secret keys on a public repository might cause security issues. :)

Reply:
Live and learn, right?